middleware.go

```diff
@@ -53,6 +54,10 @@ func createMiddleware(mw TykMiddleware) func(http.Handler) http.Handler {
 
 	return func(h http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			if txn, ok := w.(newrelic.Transaction); ok {
```

Comment:
Same here, maybe we should run it only if new relic is enabled?

Reply:
Agree.